src/components/controls/keyboard-controls.js

```diff
@@ -0,0 +1,94 @@
+var registerControls = require('../../core/controls').registerControls;
```

Comment:
We'd still have it around in a separate repo, and then we could try UX patterns like ghosting:

http://uploadvr.com/convrge-developers-introduce-improved-ghosting-a-method-for-vr-locomotion/